packages/git-commit-msg-linter/package.json

```diff
@@ -40,19 +38,9 @@
   "engines": {
     "node": ">= 8.0.0"
   },
-  "devDependencies": {
-    "eslint": "^7.28.0",
-    "eslint-config-airbnb-base": "^14.2.1",
-    "eslint-plugin-import": "^2.23.4",
-    "git-commit-msg-linter": "^4.8.3",
-    "husky": "^8.0.3",
-    "lint-staged": "^13.1.1"
-  },
   "dependencies": {
     "chalk": "^2.4.2",
-    "did-you-mean": "^0.0.1",
-    "supports-color": "^8.1.1",
```

Comment:
why are this two package deleted
and is commit-msg-linter name taken by other npm packages？

Reply:
`commit-msg-linter` rely on this two packages, not `git-commit-msg-linter`. So the package.json of `git-commit-msg-linter` can remove this two pkgs.

Reply:
Visiting https://www.npmjs.com/package/commit-msg-linter got 404.